src/LearnositySdk/Utils/Json.php

```diff
@@ -104,4 +110,63 @@ public static function isJson($val)
     {
         return !is_null(json_decode($val, true));
     }
+
+    /**
+     * This function builds up a map of float values in scientific notation and the equivalent float
+     * notations. json_encode encodes float values in scientific notation, but we want to make sure
+     * they are in float notation
+     *
+     * @param $json the $json to be encoded
+     * @param $floatMap floatMap in the current recursive call
```

Comment:
I don't think we need this argument. If you pass in:
 - a string, it returns []
- a float, it returns ["string" => "float"]
- an array [float, string], it returns ["string" => "float"] and ignore the 2nd value
- an array [float1, [float2, string]], it returns ["string1" => "float1", "string2" => "float2"]

The terminals are being captured as an array, and for any array entry, we're doing array_merge with the recursive call.